embark-theme.conf

```diff
@@ -0,0 +1,53 @@
+# Embark (https://github.com/embark-theme/kitty) {{{
```

Comment:
How would you feel about removing the fold markers? I would like to keep these files as clean as possible.